### ISSUE TITLE
Make Masterlist Error More User-Friendly

### ISSUE THREAD

**Issue by egocarib** [loot/loot#455]
After the recent hubub that users raised over a minor typo in the masterlist ([here](http://forums.bethsoft.com/topic/1519842-rel-loot-thread-19/?p=24060808) and [here](http://forums.bethsoft.com/topic/1516193-rel-loot-thread-18/?p=24060434)), I think it might be worth considering whether the error message associated with a bad conversion can be made a bit easier for users to digest. In the space of a few hours, seven users registered new accounts on Beth forums and reported this error with varying levels of panic and raucousness:
> Masterlist revision 7a43840: yaml-cpp: error at line 57, column 5: bad conversion. Rolled back to the previous revision.

I'm not saying we definitely have to modify it, but I'd like to at least hear everyone else's feelings. :smiley_cat: 
A few initial thoughts:
* It looks like this is a yaml-cpp error, so I don't know how easy it would be to modify.
* One could also argue that this kind of reaction is a good thing, as it gets our attention to fix the error.
* However, it likely causes a good amount of user confusion/frustration (probably also among many more users who didn't take the time to create a beth forums account)

I'd maybe suggest something more like this if we wanted to change it:

> Masterlist revision 7a43840: yaml-cpp: error at line 57, column 5: bad conversion. A recent change to LOOT's masterlist introduced a syntax error. LOOT will use a previous version of the masterlist instead until this error is corrected. (Syntax errors are usually corrected within a few days, but if you'd like to help move things along, you can notify LOOT developers [here](http://forums.bethsoft.com/topic/1519842-rel-loot-thread-19/).)

This feels a bit wordy, but you get the picture. Feel free to make other suggestions (or disagree with me entirely :smiley:).

**Comment by WrinklyNinja** [comment]
The `error at line 57, column 5: bad conversion` bit is the message that yaml-cpp provides, so that can't really be modified. Some additional clarification that the error isn't ushering in the apocalypse would probably help though.

Maybe

> Masterlist revision 7a43840: yaml-cpp: error at line 57, column 5: bad conversion. The latest masterlist revision contains a syntax error, LOOT is using the most recent valid revision instead. Syntax errors are generally minor and fixed within hours.

I don't think there's any need to prompt for notification, seeing as they're doing that anyway already.
